sourcegraph/about
# Use '@sourcegraph/event-logger' npm package for telemetry

Part 3 of https://github.com/sourcegraph/sourcegraph/issues/31486.

## Description
- This PR uses `@sourcegraph/event-logger` npm package instead of embeddable script.
- Also `ViewStaticPage` event includes page **path** and **UTM markers** in `argument`/`publicArgument` fields
  - UTM markers are stripped on initial load, similar to sourcegraph web
## How to test
- `yarn install`
- `yarn --cwd ./website start`
- Open any page and make sure "ViewSaticPage" event is sent
